<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Sensor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Image Enhancement</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="chkEnhancement">
            <property name="text">
             <string>Enable Enhancement</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Gamma</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsGamma">
              <property name="minimum">
               <number>30</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblGamm">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Contrast</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsContrast">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblContrast">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="labelSaturation">
              <property name="text">
               <string>Saturation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsSaturation">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblSaturation">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="lblBlackLevel">
              <property name="text">
               <string>Black Level</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsBlackLevel">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblBlack">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Exposure</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="chkAutoGain">
              <property name="text">
               <string>Auto Gain</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsGain">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>16</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblGain">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="chkAutoExposure">
              <property name="text">
               <string>Auto Exposure</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsExposure">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>4000</number>
              </property>
              <property name="sliderPosition">
               <number>8</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblExposure">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="labelTarget">
              <property name="text">
               <string>AE Target</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsAETarget">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblTarget">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QCheckBox" name="chkAntiFlicker">
              <property name="text">
               <string>Flicker Avoidence</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbPowerFreq"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbWhiteBalance">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>White Balance</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QCheckBox" name="chkAWB">
              <property name="text">
               <string>AWB</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnWB">
              <property name="text">
               <string>One Push WB</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Red Gain</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsRedGain">
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblRedGain">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Green Gain</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsGreenGain">
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblGreenGain">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Blue Gain</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsBlueGain">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblBlueGain">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Frame Control</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QCheckBox" name="cbMirrorX">
              <property name="text">
               <string>Mirror X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbMirrorY">
              <property name="text">
               <string>Mirror Y</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QCheckBox" name="cbHighspeed">
              <property name="text">
               <string>HighSpeed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Frame Delay</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="hsFrameSpeed">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="leFrameSpeed">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>B2RGB Mode</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QRadioButton" name="rbBilinear">
            <property name="text">
             <string>Bilinear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbAdaptive">
            <property name="text">
             <string>Adaptive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbHaminton">
            <property name="text">
             <string>Hamiton</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Resolution Mode</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="rbCrop">
            <property name="text">
             <string>Cropping</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbSkip">
            <property name="text">
             <string>Skipping</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbBin">
            <property name="text">
             <string>Binning</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Parameter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QRadioButton" name="rdParam0">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Group 0</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rdParam1">
                <property name="text">
                 <string>Group 1</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="btnDefault">
                <property name="text">
                 <string>Default</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnSaveParam">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
